x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_chart.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import moment from 'moment';
+import { Chart, Settings, LineSeries, CurveType, Axis } from '@elastic/charts';
+
+import { KibanaLogic } from '../../../../shared/kibana';
+
+import { X_AXIS_DATE_FORMAT, TOOLTIP_DATE_FORMAT } from '../constants';
+
+interface Props {
```

Comment:
Is a generic `Props` the convention for our React components?

Reply:
CC @JasonStoltz on this one - I think we've been doing this more and more recently in Kibana and it's because we're not really using/exporting component-specific props anymore. Interfaces or types that get reused/extended mostly live in `types.ts` files.